examples/finding_best_open_source_embedding_model/best_embedding_model_rag_app.ipynb

```diff
@@ -0,0 +1,559 @@
+    "2. Load the dataset into the PostgreSQL Docker container, which will be used throughout this tutorial.\n",
+    "\n",
+    "**Note for macOS Users:**\n",
+    "  \n",
+    "To run the script, first make it executable by running the command: `chmod +x install.sh`. Then, execute the script with: `./install.sh`.\n",
+    "\n",
+    "Let's install the necessary Python libraries for this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install pandas python-dotenv psycopg2-binary Jinja2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dataset Ingestion\n",
+    "\n",
+    "Let's setup the PostgreSQL database and install [pgai](https://github.com/timescale/pgai)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os \n",
+    "import psycopg2\n",
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "load_dotenv()\n",
+    "\n",
+    "DATABASE_CONNECTION_STRING = os.environ[\"DATABASE_CONNECTION_STRING\"]\n",
+    "\n",
+    "def connect_db():\n",
+    "    return psycopg2.connect(DATABASE_CONNECTION_STRING)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with connect_db() as conn:\n",
+    "    with conn.cursor() as cur:\n",
+    "        cur.execute(\"CREATE EXTENSION IF NOT EXISTS ai VERSION '0.5.0' CASCADE;\")\n",
+    "\n",
+    "    with conn.cursor() as cur:\n",
+    "        cur.execute(\"\"\"\n",
+    "            CREATE TABLE IF NOT EXISTS essays (\n",
+    "                id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,\n",
+    "                title TEXT NOT NULL,\n",
+    "                date TEXT,\n",
+    "                text TEXT NOT NULL\n",
+    "            );\n",
+    "          \"\"\") "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Loading the Dataset\n",
+    "\n",
+    "We'll now load Paul Graham's essays into our PostgreSQL database. The dataset consists of 215 essays with their titles, dates, and full text content. We'll verify the successful ingestion by displaying the first essay from the database."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with connect_db() as conn:\n",
+    "    with conn.cursor() as cur:\n",
+    "        # Copy data from the CSV file stored in your docker container into the 'essays' table, \n",
+    "        # starting from the second column\n",
+    "        cur.execute(\"\"\"\n",
+    "            COPY essays (title, date, text)\n",
+    "            FROM PROGRAM 'cut -d, -f2- /tmp/paul_graham_essays.csv'\n",
+    "            DELIMITER ','\n",
+    "            CSV HEADER;\n",
+    "        \"\"\")\n",
```

Comment:
This doesn't seem to work with the internal csv I could find? It also doesn't work with any other csv representation of the essays. I could find. Shouldn't the install.sh download the correct representation or we just add the file to the repo?